TShock.Modifications.Platform/PlatformModification.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Mono.Cecil.Cil;
+using OTAPI.Patcher.Engine.Modification;
+
+namespace Mintaka.Modifications.Platform
+{
+	/// <summary>
+	/// This modification will make OTAPI initialized with right platform rather than Windows platform.
+	/// This should fix different world path OTAPI used on macOS
+	/// In addition, the console input will also be fixed.
+	/// </summary>
+	public class UnicodeInputModification : ModificationBase
```

Comment:
I'm approving this review, but I would generally prefer if @DeathCradle or @tylerjwatson looked at this. I'd hold out for either of their approvals as they are the mad scientists.